README.md

```diff
@@ -252,6 +252,51 @@ rescue WebAuthn::Error => e
+
+>When creating a public key credential or requesting an authentication assertion, a WebAuthn Relying Party can request the use of a set of extensions. These extensions will be invoked during the requested operation if they are supported by the client and/or the WebAuthn Authenticator. The Relying Party sends the client extension input for each extension in the get() call (for authentication extensions) or create() call (for registration extensions) to the client. [[source](https://www.w3.org/TR/webauthn-2/#sctn-extensions)]
+
+Extensions can be requested in the initiation phase in both Credential Registration and Authentication ceremonies by adding the extension parameter when generating the options for create/get.
+
+```ruby
+# Credential Registration
+creation_options = WebAuthn::Credential.options_for_create(
+  user: { id: user.webauthn_id, name: user.name },
+  exclude: user.credentials.map { |c| c.webauthn_id },
+  extensions: { appidExclude: domain.to_s }
+)
+
+# OR
+
+# Credential Authentication
+options = WebAuthn::Credential.options_for_get(
+  allow: user.credentials.map { |c| c.webauthn_id },
+  extensions: { appid: domain.to_s }
+)
+```
+
+After `options` are send back to the browser:
+
+>The client performs client extension processing for each extension that the client platform supports, and augments the client data as specified by each extension, by including the extension identifier and client extension output values.
+
+>For authenticator extensions, as part of the client extension processing, the client also creates the CBOR authenticator extension input value for each extension (often based on the corresponding client extension input value), and passes them to the authenticator in the create() call (for registration extensions) or the get() call (for authentication extensions).
+
+>The authenticator, in turn, performs additional processing for the extensions that it supports, and returns the CBOR authenticator extension output for each as specified by the extension. Part of the client extension processing for authenticator extensions is to use the authenticator extension output as an input to creating the client extension output. [[source](https://www.w3.org/TR/webauthn-2/#sctn-extensions)]
+
+You can check the values returned for each extension by calling `client_extension_outputs` and `authenticator_extension_outputs` respectively.
+For example, following the initialization phase for the Credential Authentication ceremony specifyied in the above example:
+
+```ruby
+webauthn_credential = WebAuthn::Credential.from_get(params[:publicKeyCredential])
+
+webauthn_credential.client_extension_outputs #=> <ActionController::Parameters {"appid"=>true} permitted: false>
+webauthn_credential.authenticator_extension_outputs #=> nil
+```
+
+The list of all current implemented extensions can be found [here](https://w3c.github.io/webauthn/#sctn-defined-extensions).
```

Comment:
Here, I added the link to the [editor’s draft documentation](https://w3c.github.io/webauthn/), which is different from the one we are using in this file. That is because the editor's draft contains the updated list of extensions currently implemented after a lot were removed from the spec in https://github.com/w3c/webauthn/issues/1386.

Reply:
Make sense.

I updated so that we link to the two lists, the "last published" and the "draft" one.